README.md

```diff
@@ -201,6 +201,70 @@ this.shepherdService.addSteps([
 ]);
 ```
 
+## Buttons
+
+In Shepherd, you can have as many buttons as you want inside a step. You can build an object with some premade buttons, making it easier to manipulate and insert in new steps. Buttons by default accept three different types: back, cancel, next.
+
+```js
+const builtInButtons = {
+  cancel: {
+    classes: "cancel-button",
+    text: "cancel",
+  },
+  next: {
+    classes: "next-button",
+    text: "Next",
+    type: "next"
+  },
+  back: {
+    classes: "back-button",
+    secondary: true,
+    text: "Back",
+    type: "back"
+  }
+};
+```
+
+In this simple example, we have three buttons: each one with different types and classes. 
+
+**⚠️ You can't set up a type and an action at the same time inside a button**. 
+
+There are some techniques to make certain buttons perform the actions you want on the component. In Angular, you can use Decorators to make some actions work by events. For example, you can listen for document clicks (mouse events) and check what the user is doing.
+
+First, you need to import HostListener from @angular/core. 
```

Comment:
I don't think we should document this stuff. You can pass `action` to your buttons for custom actions, no need to do all this, right?

Reply:
I agree with you, but if you use a separated js file from your component, you can't access different services. For custom actions inside angular components this is a workaround. Not sure if it should be in shepherd readme, let me know. I may write more about action

Reply:
I'm saying that the `action` needs to be a function. Where the action comes from can vary based on the framework and setup of your app.